tests-integration/reference-model/attributemodel/src/Morphir/Attribute/Model/Sensitivity.elm

```diff
@@ -0,0 +1,5 @@
+module Morphir.Attribute.Model.Sensitivity exposing (..)
+
+-- @docs Sensitivity
+
+type Sensitivity = MNPI | PII
```

Comment:
Sorry, I know I haven't mentioned this before but could you add all the remaining sensitivity classifications from here: https://bigid.com/blog/sensitive-information-guide/